cip-drep-metadata/README.md

```diff
@@ -0,0 +1,231 @@
+- donations
+- expenses
+- any incentive program
+
+Therefore there will be a `payment_address` field in the metadata where such payments could be sent. This makes such an address public and payments to DReps transparent. 
+
+`username`
+- Compulsory
+- A very short freeform text field. Limited to 80 characters.
+- This SHOULD NOT support markdown text styling.
+- Authors MUST use this field for their profile name/ username.
+- Authors SHOULD attempt to make this field unique.
+- Authours SHOULD avoid crass language.
+
+The compulsory nature of this field was controversial because the `username`s cannot be made unique and therefore are open to abuse (by e.g. copycats). However this is not a reason to not include a `username` it a reason for people reviewing governance actions to properly check the whole profile of a DRep before delegating to them. A `username` MUST be included because it is a human readable identifier, it is the property that people reviewing DReps will most likely identify that DRep by even in the presence of copycats.
+
+`picture`
+- Optional 
+- A base 64 encoded profile picture
+- Moderation of this image must be handled on the client side to comply with their TOS
+- This SHOULD be treated as the profile picture of the individual
+
+`objectives`
+- Optional
+- A freeform text field with a maximum of 1000 characters
+- A short description of what the person believes nd what they want to achieve as a DRep
+
+`motivation`
+- Optional
+- A freeform text field with a maximum of 1000 characters
+- A short description of why they want to be a DRep, what personal and professional experiences have they had that have driven them to 
+
+`qualifications`
+- Optional
+- A freeform text field with a maximum of 1000 characters 
+- A space to list the qualifications that the subject of this metadata has that are relevant to being a DRep
+
+`identity`
+- Compulsory
+- A link to a social media profile where the person must reference their DRep ID in their profile.
+- This will be used by people reviewing this DRep to prove and verify that the person described in the metadata is the same as the person who set up the social media profile.
+- Tooling providers SHOULD warn people that none of the information is verified by the tool and they should DYOR
+- Tooling providers making metadata SHOULD provide some information about why this is important.
+
+PKI cryptographic solutions were also considered in the place of the current solution however they were generally considered too complex to implement for minimum viable tooling. There were also other issues with cryptographic forms of identification for MVPs:
+1. solutions that involve a public/private key setup still require people verifying the identity of a DRep to know what the authentic pulic key is.
+2. specifying the use of a verification service such as [Keybase](https://keybase.io/) would lead to centralisation and therefore reliance on a 3rd party for DReps.
+
+`links`
```

Comment:
why not reuse the References or externalUpdates from CIP100?

Reply:
lets discuss this.

Reply:
Ok, so the decision was that References should be used for all external links and that we can extend CIP-100 to add a @type called "identity" and possibly one called "link".

Reply:
@Ryun1 I have made this [change](https://github.com/Thomas-Upfield/CIPs/blob/drep-metadata/cip-drep-metadata/README.md#references)